BUILD_INSTRUCTIONS_FOR_LINUX.md

```diff
@@ -0,0 +1,39 @@
+# Build Instructions for Linux
+
+## Required Libraries
+*   Python2
```

Comment:
In theory we shouldn't depend on Python2. Could you please help point out where we require Py2?

Reply:
You are right python version does not matter. I'll update it as well.